frontend_server/lib/server.dart

```diff
@@ -261,6 +246,14 @@ Future<int> starter(
   }
 
   if (options['train']) {
+    final String sdkRoot = options['sdk-root'];
```

Comment:
This chunk of code was reverted by @aam because it breaks the roll. 

You would need to keep this `if` empty.

Reply:
Disregard this comment. @jensjoha has fixed and relanded his snapshot training patch.